waltz-tools/src/main/java/com/wepay/waltz/tools/cluster/ClusterCli.java

```diff
@@ -241,6 +236,16 @@ protected void processCmd(CommandLine cmd) throws SubCommandFailedException {
                     );
                 }
 
+                // Step3: Build Server-Storage connectivity validations
```

Comment:
any particular reason for moving this?

Reply:
Just for consistency. To have implementation of one step after the other. Easy to understand.

Reply:
Oh, okay. The reason I added was to have parallelism in fetching information from the server, rather than doing one-by-one for they are independent.

Reply:
aah okay. will revert that change.